src/controllers/processors/CompanyNameProcessor.ts

```diff
@@ -0,0 +1,59 @@
+import { SessionKey } from 'ch-node-session-handler/lib/session/keys/SessionKey';
+import { ISignInInfo } from 'ch-node-session-handler/lib/session/model/SessionInterfaces';
+import { CompanyProfile } from 'ch-sdk-node/dist/services/company-profile/types';
+import Resource from 'ch-sdk-node/dist/services/resource';
+import { Request } from 'express';
+import { OK } from 'http-status-codes';
+import { inject } from 'inversify';
+import { provide } from 'inversify-binding-decorators';
+import { FormActionProcessor } from './FormActionProcessor';
+
+import { PenaltyIdentifier } from 'app/models/PenaltyIdentifier';
+import { CompaniesHouseSDK, OAuth2 } from 'app/modules/Types';
+
+
+export const SESSION_NOT_FOUND_ERROR: Error = new Error('Session Expected but was undefined');
+export const TOKEN_MISSING_ERROR: Error = new Error('Token was expected');
```

Comment:
Can we expand on this error message, for example 'Access token missing from session'